packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  withExperimentalFeatures,
+  visitSettings,
+  addCustomFont,
+  removeCustomFont,
+  getFontList,
+  getSelectedFont,
+} from '@web-stories-wp/e2e-test-utils';
+
+const FONT_BASE_URL = `${process.env.WP_BASE_URL}/wp-content/e2e-assets`;
+const OPEN_SANS_CONDENSED_LIGHT = 'Open Sans Condensed Light';
+const OPEN_SANS_CONDENSED_LIGHT_URL = `${FONT_BASE_URL}/OpenSansCondensed-Light.ttf`;
+const OPEN_SANS_CONDENSED_BOLD = 'Open Sans Condensed Bold';
+const OPEN_SANS_CONDENSED_BOLD_URL = `${FONT_BASE_URL}/OpenSansCondensed-Bold.ttf`;
+const OPEN_SANS_CONDENSED_LIGHT_ITALIC_URL = `${FONT_BASE_URL}/OpenSansCondensed-LightItalic.ttf`;
+
+const findByUrl = (arr, val) => {
+  const found = arr.filter((o) => o.url === val);
+  if (found) {
+    return found[0];
+  }
+  return [];
+};
+
+const removeAllFonts = async () => {
+  await visitSettings();
+  let fonts = await getFontList();
```

Comment:
I don't think this needs to be a let, we can just have two consts here. I don't see the value of reusing the variable here.

Reply:
Updated 

https://github.com/GoogleForCreators/web-stories-wp/pull/10918/files/9bf20a7d87aa6f3b690d8ff6a3aa17924d647bc5#diff-15102bcca0cb4bffec2cea43a88ada5c191041b40f7e784266ac7b5cd2161a1dR18